pubspec.yaml

```diff
@@ -30,15 +28,15 @@ dependencies:
   wave: 0.2.2
   texture_rgba_renderer:
     git: https://github.com/h3x4d3c1m4l/flutter_texture_rgba_renderer
-  lottie: 3.1.1
+  lottie: 3.1.2
   font_awesome_flutter: 10.7.0
   smooth_scroll_multiplatform: 1.0.8
   widgetbook: 3.7.1
   widgetbook_annotation: 3.1.0
-  flex_color_picker: 3.4.1
+  flex_color_picker: 3.5.0
 
   # Printscreen/Image processing
-  screenshot: 2.3.0
+  screenshot: 3.0.0
```

Comment:
Anything new in v3?

Reply:
Nothing at all except compatibility! This is simply a semver bump because the change for Flutter 3.22 makes it incompatible with previous versions.